examples/mpi/runner/pipeline.json

```diff
@@ -0,0 +1,132 @@
+{
```

Comment:
@alesgenova This `pipeline.json` is just like the basic `pipeline.json`, except it includes the locations key, and the path to the singularity images instead of docker image names (which we, at some point, can probably automatically convert between them).